rules/windows/discovery_net_command_system_account.toml

```diff
@@ -2,13 +2,12 @@
 creation_date = "2020/03/18"
 ecs_version = ["1.5.0"]
 maturity = "production"
-updated_date = "2020/08/03"
+updated_date = "2020/08/12"
 
 [rule]
 author = ["Elastic"]
 description = """
-Identifies the SYSTEM account using the Net utility. The Net utility is a component of the Windows operating system. It
-is used in command line operations for control of users, groups, services, and network connections.
+Identifies the SYSTEM account using an account discovery utility. This could be a sign of discovery post successful privileges elevation.
```

Comment:
Minor suggestion to try and help with readability.

```suggestion
Identifies the SYSTEM account using an account discovery utility. This could be a sign of discovery activity after an adversary has achieved privilege escalation.
```